models/db/question.model.js

```diff
@@ -58,6 +58,11 @@ module.exports = function question(sequelize, Sequelize, schema) {
         common: {
             type: Sequelize.BOOLEAN,
         },
+        isIdentifying: {
+            type: Sequelize.BOOLEAN,
```

Comment:
This is where you'd set a property of `field: 'is_identifying'`, so you can retain camelCase elsewhere in JavaScript code.